src/RpcServer/RpcError.cs

```diff
@@ -0,0 +1,101 @@
+        //public static readonly RpcError UnknownIterator = new(-109, "Unknown iterator");
+
+        public static readonly RpcError InsufficientFundsWallet = new(-300, "Insufficient funds in wallet");
+        public static readonly RpcError WalletFeeLimit = new(-301, "Wallet fee limit exceeded", "The necessary fee is more than the Max_fee, this transaction is failed. Please increase your Max_fee value.");
+        //public static readonly RpcError NoOpenedWallet = new(-302, "No opened wallet");
+        //public static readonly RpcError WalletNotFound = new(-303, "Wallet not found");
+        //public static readonly RpcError WalletNotSupported = new(-304, "Wallet not supported");
+
+        public static readonly RpcError AccessDenied = new(-400, "Access denied");
+
+        public static readonly RpcError VerificationFailed = new(-500, "Inventory verification failed");
+        public static readonly RpcError InvalidSignature = new(-501, "Invalid signature");
+        public static readonly RpcError InsufficientFunds = new(-502, "Insufficient funds for fee");
+        public static readonly RpcError InvalidContractVerification = new(-503, "Invalid contract verification function");
+        public static readonly RpcError AlreadyExists = new(-504, "Inventory already exists");
+        //public static readonly RpcError MempoolCapReached = new(-505, "Memory pool capacity reached");
+        //public static readonly RpcError AlreadyInPool = new(-506, "Already in pool");
+        //public static readonly RpcError InsufficientNetworkFee = new(-507, "Insufficient network fee");
+        //public static readonly RpcError PolicyFailed = new(-508, "Policy check failed");
+        //public static readonly RpcError InvalidScript = new(-509, "Invalid transaction script");
+        //public static readonly RpcError InvalidAttribute = new(-510, "Invalid transaction attribute");
+        //public static readonly RpcError InvalidSize = new(-511, "Invalid inventory size");
+        //public static readonly RpcError ExpiredTransaction = new(-512, "Expired transaction");
+
+        public static readonly RpcError OracleRequestFinished = new(-600, "Oracle request already finished");
+        public static readonly RpcError OracleRequestNotFound = new(-601, "Oracle request not found");
+        public static readonly RpcError OracleNotDesignatedNode = new(-602, "Not a designated oracle node");
+        //public static readonly RpcError OracleDisabled = new(-603, "Oracle service disabled");
+
+        public static readonly RpcError UnsupportedState = new(-700, "Old state not supported");
+        public static readonly RpcError InvalidProof = new(-701, "Invalid state proof");
+        //public static readonly RpcError SessionsDisabled = new(-702, "State iterator sessions disabled");
+        //public static readonly RpcError ExecutionFailed = new(-703, "Contract execution failed");
```

Comment:
Please, fix the codes according to https://github.com/neo-project/proposals/pull/156.

Reply:
We have:

```#
        public static readonly RpcError BadRequest = new(-32700, "Bad request");
        public static readonly RpcError InvalidRequest = new(-32600, "Invalid request");
        public static readonly RpcError MethodNotFound = new(-32601, "Method not found");
        public static readonly RpcError InvalidParams = new(-32602, "Invalid params");
```
that are not defined in the protocol, how does neogo process them?

Reply:
They're defined.
> Compliant server implementations can use error codes defined by JSON-RPC
specification (from -32700 to -32600) for any calls when they're
appropriate.

https://www.jsonrpc.org/specification